inc/sai.h

```diff
@@ -140,6 +142,8 @@ typedef enum _sai_api_t
     SAI_API_MY_MAC           = 45, /**< sai_my_mac_api_t */
     SAI_API_IPSEC            = 46, /**< sai_ipsec_api_t */
     SAI_API_GENERIC_PROGRAMMABLE = 47, /**<sai_generic_programmable_t */
+    SAI_API_ARS              = 48, /**< sai_ars_api_t */
```

Comment:
Are you planning to amend SAI pipeline diagram to indicate how the new tables/objects fit in the existing pipeline model?

Reply:
This feature appears just another detail of ECMP, I feel it is not necessary.

Reply:
Even if don't update the existing pipeline diagrams, I think it would help to add a logical pipeline diagram in this PR itself.

Reply:
Added a diagram in the new PR